packages/data/src/world/mm/great_bay_temple.yml

```diff
@@ -230,10 +230,10 @@
     "Great Bay Temple Boss Pot 2": "true"
     "Great Bay Temple Boss Pot 3": "true"
     "Great Bay Temple Boss Pot 4": "true"
-    "Great Bay Temple Boss Pot Underwater 1": "underwater_walking"
-    "Great Bay Temple Boss Pot Underwater 2": "underwater_walking"
-    "Great Bay Temple Boss Pot Underwater 3": "underwater_walking"
-    "Great Bay Temple Boss Pot Underwater 4": "underwater_walking"
+    "Great Bay Temple Boss Pot Underwater 1": "underwater_walking || (has_mask_blast && can_dive_big)" # todo blast+dive should be a trick
+    "Great Bay Temple Boss Pot Underwater 2": "underwater_walking || (has_mask_blast && can_dive_big)" # todo blast+dive should be a trick
+    "Great Bay Temple Boss Pot Underwater 3": "underwater_walking || (has_mask_blast && can_dive_big)" # todo blast+dive should be a trick
+    "Great Bay Temple Boss Pot Underwater 4": "underwater_walking || (has_mask_blast && can_dive_big)" # todo blast+dive should be a trick
```

Comment:
Not to mention the logic could have you do these without having the means to defeat Gyrog... You are definitely not getting these while Gyorg is present. These with scale should be placed in the After Boss region so that you can defeat Gyorg with FD and then get them.

Reply:
Sounds good, but if you come here without gyorg's soul, the pots would be out of logic using a dive. is that okay?